utils/dbutils.py

```diff
@@ -119,33 +119,33 @@ def getVFATFilter(vfatList):
     strRetFilter = " WHERE ("
     for idx, vfatID in enumerate(vfatList):
         if idx == 0:
-            strRetFilter += " data.VFAT3_SER_NUM='{0}'".format(str(hex(vfatID)).strip('L'))
+            strRetFilter += " data.VFAT3_SER_NUM='0x{:x}'".format(vfatID)
         else:
-            strRetFilter += " OR data.VFAT3_SER_NUM='{0}'".format(str(hex(vfatID)).strip('L'))
+            strRetFilter += " OR data.VFAT3_SER_NUM='0x{:x}'".format(vfatID)
             pass
         pass
     strRetFilter += " )\n"
 
     return strRetFilter
 
-def joinOnVFATSerNum(vfatList, dfGemView):
+def joinOnVFATSerNum(vfatList, dfGEMView):
     """
     Creates a dataframe object from vfatList with keys 'vfat3_ser_num' and 'vfatN'.
-    Then it joins this dataframe with dfGemView using the 'vfat3_ser_num'.
+    Then it joins this dataframe with dfGEMView using the 'vfat3_ser_num'.
 
     vfatList - A list of vfat ChipID's ordered by vfat position (sw)
-    dfGemView - A pandas dataframe containing the column name 'vfat3_ser_num'
+    dfGEMView - A pandas dataframe containing the column name 'vfat3_ser_num'
     """
 
-    if 'vfat3_ser_num' in dfGemView.columns:
+    if 'vfat3_ser_num' in dfGEMView.columns:
         dfVFATPos = pd.DataFrame(
                     {   'vfatN':[vfat for vfat in range(24)], 
-                        'vfat3_ser_num':[str(hex(id)).strip('L') for id in vfatList]}
+                        'vfat3_ser_num':["0x{:x}".format(id) for id in vfatList]}
                 )
 
-        dfGemView = pd.merge(dfVFATPos, dfGemView, on='vfat3_ser_num', how='outer')
+        dfGEMView = pd.merge(dfVFATPos, dfGEMView, on='vfat3_ser_num', how='outer')
     else:
-        printYellow("column 'vfat3_ser_num' not in input dataframe columns: {0}".format(dfGemView.columns))
+        printYellow("column 'vfat3_ser_num' not in input dataframe columns: {0}".format(dfGEMView.columns))
         pass
 
-    return dfGemView
+    return df_gemView
```

Comment:
You've got a mismatch here, no?
At least as I see it, the thing being returned (`df_gemView`) is never declared previously

Reply:
The dangers of highlighting a section and using sed replacement.